File: Assets/Scripts/Finish.js
#pragma strict


function OnTriggerEnter2D (obj : Collider2D){
	
	if(obj.name == "Player"){
		
//		var cam : CameraPosition = GetComponent(CameraPosition);
//		cam.target = camera.transform;
		//TODO: Disable controll script
		CameraPosition.target = transform;
		//TODO: PLay animation
	
	}

}

File: Assets/Scripts/Finish.cs
using UnityEngine;
using System.Collections;

public class Finish : MonoBehaviour {
	private AudioSource source;
	public AudioClip levelCleared;
	Animator anim;

	void Start(){
		anim = GetComponent<Animator> ();
		source = GetComponent<AudioSource>();
	}

	void OnTriggerEnter2D (Collider2D obj){
		
		if(obj.name == "Player"){
			//mute the theme music
			Camera.main.GetComponent<AudioSource>().mute = true;
			// play the level cleared theme
			source.clip = levelCleared;
			source.Play();
			CameraPosition.target = transform;
			PlayerScript.finished = true;
			PlayerScript.isMoving = false;
			CanvasController.theEnd = true;
			CanvasController.clearedLevel = true;
			anim.SetTrigger("Finished");
			
//			
		}
		
	}
}
